cmd/bootstrap/transit/cmd/pull.go

```diff
@@ -0,0 +1,133 @@
+package cmd
+
+import (
+	"context"
+	"fmt"
+	"path/filepath"
+	"time"
+
+	"github.com/spf13/cobra"
+
+	bootstrap "github.com/onflow/flow-go/cmd/bootstrap/cmd"
+	"github.com/onflow/flow-go/ledger/complete/wal"
+	model "github.com/onflow/flow-go/model/bootstrap"
+	"github.com/onflow/flow-go/model/flow"
+)
+
+var (
+	flagToken      string
+	flagNodeRole   string
+	flagBucketName string
+)
+
+// pullCmd represents a command to pull keys and metadata from the Google bucket
+var pullCmd = &cobra.Command{
+	Use:   "pull",
+	Short: "Fetch keys and metadata from the transit server",
+	Long:  `Fetch keys and metadata from the transit server`,
+	Run:   pull,
+}
+
+func init() {
+	rootCmd.AddCommand(pullCmd)
+	addPullCmdFlags()
+}
+
+func addPullCmdFlags() {
+	pullCmd.Flags().StringVar(&flagToken, "token", "", "token provided by the Flow team to access the Transit server")
+	pullCmd.Flags().StringVar(&flagNodeRole, "role", "", `node role (can be "collection", "consensus", "execution", "verification" or "access")`)
+	pullCmd.Flags().StringVar(&flagBucketName, "bucket", "", "the name of the Google Bucket provided by the Flow team")
```

Comment:
Couple of things:
1. We should avoid mentioning `Google Bucket` since it is an implementation detail plus people might get ideas to snoop around the google public buckets we expose.
2. The `token` was suppose to be the bucket name. Is there a reason why we have token different than the bucket? (the token was nothing but the name of the bucket)

Reply:
The `token` and `flowBucket` was used by the transit script https://github.com/onflow/flow-go/blob/master/cmd/bootstrap/transit/main.go#L66. So wasn't sure if there was subdirectory in the bucket that was given as a `keyDir` (in the above link).

Reply:
My vote is to keep these names as is to minimize the changes needed in the documentation and operator flow.